### DISCUSSION TITLE
How to start the Mongodb or MySQL DB  Extensions Service in Theia IDE

### DISCUSSION THREAD

**Discussion by Gopikrishna-GitHB** [eclipse-theia/theia#12118] [Q&A]
@msujew We are facing issue while initiating the service of MongoDb/MySQL extensions After installation in Theia IDE. I am attaching the screenshot of the error receiving while starting the service. 
If there is any steps to fix it please update it. Thanks in Advance.

![MicrosoftTeams-image (14)](https://user-images.githubusercontent.com/85285581/215098727-d400c411-75d4-4a0b-8f39-c03597eaaf6c.png)

**Comment by msujew** [comment]
Hey @Gopikrishna-GitHB,

this isn't really related to Theia. as this seems to be purely about the interaction of MongoDB/MySQL within your docker container? You'll need to install your dev-prerequisites into your image yourself, Theia (and neither the MongoDB extension) won't do that for you.
